rllib/env/vector_env.py

```diff
@@ -237,11 +237,6 @@ def vector_step(self, actions):
         obs_batch, rew_batch, done_batch, info_batch = [], [], [], []
         for i in range(self.num_envs):
             obs, r, done, info = self.envs[i].step(actions[i])
-            if not np.isscalar(r) or not np.isreal(r) or not np.isfinite(r):
```

Comment:
?

Reply:
Yes and no. :) We are already checking this now via the new env checking module before.
Just wanted to remove redundancy here. Otherwise one could argue that we need to add this check to all our other env types as well.